modules/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovySecurityTests.java

```diff
@@ -123,6 +123,12 @@ public void testEvilGroovyScripts() throws Exception {
         }
     }
 
+    public void testGroovyScriptsThatThrowErrors() throws Exception {
+        assertFailure("assert false, \"msg\";", AssertionError.class);
+        assertFailure("def foo=false; assert foo;", AssertionError.class);
+        assertFailure("def bar=false; assert bar, \"msg2\"", NoClassDefFoundError.class);
```

Comment:
Sure, but that was before we knew that we would need to add a permission for `InvokeHelper` too? So maybe we should just not add it since it will not help with getting asserts to work? Sorry.